spec/models/performance_statistics_spec.rb

```diff
@@ -91,12 +91,10 @@
       create_list(:application_choice, 2, application_form: rejected_form, status: 'rejected')
       create_list(:application_choice, 2, application_form: declined_form, status: 'declined')
       create_list(:application_choice, 2, application_form: conditions_not_met_form, status: 'conditions_not_met')
-
```

Comment:
Where possible, we tend to separate the arrange/act/assert parts of specs (where distinct!) with a blank line. I can understand the removal below, but I think this bit is more readable with the line left in?